generator/models.ts

```diff
@@ -0,0 +1,26 @@
+enum ScopeType {
+    None = 0,
+    Unknown = 1 << 0,
```

Comment:
>Unknown [](start = 4, length = 7)

Is there an example of "Unknown" type?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/f6e96392dd7f1069b6d600cfda549bf786d009b0/specification/resources/resource-manager/Microsoft.Authorization/stable/2018-03-01/policyAssignments.json#L37

Whenever the swagger spec specifies something generic like '{scope}' without any way of inferring what the scope can be.